.github/scripts/approve-rule.mjs

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env zx
+import 'zx/globals';
+import assert from 'assert';
+import moment from 'moment';
+
+import {
+  config,
+  cloneWcagActRules,
+  createOrCheckoutBranch,
+  commitAndPush
+} from './commons.mjs';
+
+const w3cDataFormat = 'D MMMM YYYY';
+const isoDateFormat = 'YYYY-MM-DD';
+
+assert(typeof argv.ruleId === 'string', 'Expected --ruleId to be set');
+assert(argv.ruleId.length === 6, 'Expected --ruleId to be 6 characters long');
+assert(typeof argv.branch === 'string', 'Expected --branch to be set');
+
+if (!argv['skip-clone']) {
+  await cloneWcagActRules(config);
+}
+
+await createOrCheckoutBranch(config, argv.branch);
+await generateApprovedRulePages(config, argv.ruleId);
+await updateRuleVersionsYaml(config, argv.ruleId);
+await approveTestCaseJson(config, argv.ruleId);
+await commitAndPush(config, `Set ${argv.ruleId} to approved`);
```

Comment:
This script basically automated the workflow documented here:

https://github.com/w3c/wcag-act-rules#adding-new-approved-rules